### PR TITLE
[DRAFT] OU-405: Adjust the monitoring CI to publish images in quay

### PR DESCRIPTION
**Pull request by zhuje**

### CODE REVIEW

File: ci-operator/config/openshift/troubleshooting-panel-console-plugin/openshift-troubleshooting-panel-console-plugin-release-0.2.yaml [hunk 1]
```diff
@@ -20,7 +20,6 @@ images:
 promotion:
   to:
   - excluded_images:
-    - troubleshooting-panel-console-plugin
```
Comment (PeterYurkovich) [inline ci-operator/config/openshift/troubleshooting-panel-console-plugin/openshift-troubleshooting-panel-console-plugin-release-0.2.yaml:23]: This looks like it should have been the test image I think

Comment (zhuje) [reply ci-operator/config/openshift/troubleshooting-panel-console-plugin/openshift-troubleshooting-panel-console-plugin-release-0.2.yaml:23]: Yeah, I think the same.

**Pull request closed without merging**